velox/common/serialization/Serializable.h

```diff
@@ -314,6 +320,25 @@ class ISerializable {
   }
 
   virtual ~ISerializable() = default;
+
+ private:
+  template <
+      class T,
+      typename = std::enable_if_t<std::is_base_of_v<ISerializable, T>>>
+  static auto deserializeAsUniquePtr(
+      const folly::dynamic& obj,
+      void* context = nullptr) {
```

Comment:
'context' appears unused here. Is this intentional? Shouldn't it be used?

Reply:
Hmm... Shouldn't it be used? Since this is a generic API, I expect it to support deserializers that require additional context.